AssetManagementTestProject/TestCases/US303LogoutTest.cs

```diff
@@ -0,0 +1,51 @@
+﻿using AssetManagementTestProject.PageObj;
+using AssetManagementTestProject.TestSetup;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+
+namespace AssetManagementTestProject.TestCase;
+
+[TestFixture]
+public class US303LogoutTest : NUnitWebTestSetup
+{
+    [TestCase(Constant.ADMIN_USERNAME_HN, Constant.ADMIN_PASSWORD_HN)]
+    [TestCase(Constant.STAFF_USERNAME, Constant.STAFF_PASSWORD)]
+    public void UserCanLogoutSuccessfully(string userName, string password)
+    {
+        LoginPage?.Login(userName, password);
+        DriverBaseAction?.WaitToBeVisible(HomePage.HeaderHomePage);
+        Asserter?.AssertElementIsDisplayed(HomePage.HeaderHomePage);
+
+        HomePage?.SelectLogout();
+        LogoutPopup?.LogOutOfPage();
+        Asserter?.AssertElementsAreDisplayed(LoginPage.GetLoginPageElement());
```

Comment:
This assertion will be passed even if a blank page is displayed. Please check the Get method again.